gradle/libs.versions.toml

```diff
@@ -38,7 +37,6 @@ ktor-client-core = { module = "io.ktor:ktor-client-core", version.ref = "ktor" }
 ktor-client-logging = { module = "io.ktor:ktor-client-logging", version.ref = "ktor" }
 ktor-client-mock = { module = "io.ktor:ktor-client-mock", version.ref = "ktor" }
 ktor-serialization-kotlinx-json = { module = "io.ktor:ktor-serialization-kotlinx-json", version.ref = "kotlinSerialization" }
-logback-classic = { module = "ch.qos.logback:logback-classic", version.ref = "logback" }
```

Comment:
Removed logback as there was a dependabot PR to update this and it isn't being used. I don't want to approve a request for something that isn't used.